code/go/0chain.net/smartcontract/storagesc/clientpool.go

```diff
@@ -0,0 +1,722 @@
+package storagesc
```

Comment:
Rather than block copy code from somewhere. see if you can come up with a way of using common code wherever possable. So bugs will only have to be fixed in one place. For example consider [function.go](https://github.com/0chain/0chain/blob/966d7479999ac15ce1642e029154e73c95405ba3/code/go/0chain.net/smartcontract/storagesc/functions.go).